types/src/transaction/transaction.rs

```diff
@@ -696,6 +879,12 @@ impl PartialHashing for Transaction {
     }
 }
 
+impl PartialHashing for Order {
+    fn hash_partially(&self, _tag: Tag, _cur: &AssetTransferInput, _is_burn: bool) -> Result<H256, HashingError> {
+        Ok(blake256(&self.rlp_bytes()))
```

Comment:
The hash value of orders should be fixed regardless of any arguments.
Is this okay to do so? @sgkim126 @joojis

Reply:
Should `Order` implement PartialHashing?
Which code is using it?

Reply:
@joojis The unlock script on an order should sign the order, not the transaction itself. It allows to give an order to other people (relayers or takers) with an proof of ownership, and temporal approval.